homeassistant/components/electric_kiwi/icons.json

```diff
@@ -0,0 +1,18 @@
+{
+  "entity": {
+    "sensor": {
+      "total_running_balance": {
+        "default": "mdi:currency-usd"
+      },
+      "total_current_balance": {
+        "default": "mdi:currency-usd"
```

Comment:
I assume this means being able to display different currency symbols per language? This is great!

Reply:
No, in this case translation rather means that the state is translated into a icon (https://developers.home-assistant.io/blog/2024/01/19/icon-translations/).